server/src/main/resources/application-qa.properties

```
@@ -10,7 +10,6 @@ kitu.kotoutumiskoulutus.koealusta.scheduling.enabled=true
 kitu.kotoutumiskoulutus.koealusta.scheduling.import.schedule=FIXED_DELAY|3600s
 
 kitu.oppijanumero.username=koto-rekisteri
-kitu.oppijanumero.callerid=1.2.246.562.24.77101904300
```

Pitäiskö tässä olla joku arvo, esim:
```suggestion
kitu.oppijanumero.callerid=1.2.246.562.24.77101904300.koto-rekisteri
kitu.oppijanumero.casUrl=https://virkailija.testiopintopolku.fi/cas
```